releases/release-1.30/release-notes/maps/pr-118756-map.yaml

```
@@ -0,0 +1,3 @@
+pr: 118756
+releasenote:
+  text: ""
```

I believe this should work for this PR
```suggestion
  text: "NONE"
```